modules/dnn/src/layers/slice_layer.cpp

```diff
@@ -58,6 +58,30 @@ namespace cv
 namespace dnn
 {
 
+void sliceRangesFromShape(const MatShape& inpShape, int& axis, std::vector<std::vector<cv::Range>>& sliceRanges)
```

Comment:
`error: '>>' should be '> >' within a nested template argument list`

(3.4 branch uses C++98)

Reply:
Yes, I know, thanks. I forget to set c++ standard sometimes.